### PR TITLE
Airbnb linter

### PR DESCRIPTION
**Pull request by 2940cristian**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+var OFF = 0, WARN = 1, ERROR = 2;
+
+module.exports = exports = {
+    "env": {
+        "es6": true
```
Comment (scinos) [inline .eslintrc.js]: I think this should include `"browser": true`

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+module.exports = exports = {
+    "env": {
+        "es6": true
+    },
+
+    "extends": ["react-app", "airbnb"],
+
+    "rules": {
+        "eol-last": 0,
```
Comment (scinos) [inline .eslintrc.js]: You can also specify `"eol-last": "off"` (https://eslint.org/docs/user-guide/configuring#configuring-rules)

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+var OFF = 0, WARN = 1, ERROR = 2;
```
Comment (scinos) [inline .eslintrc.js]: Unused

File: package.json [hunk 4]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "eslint-config-airbnb": "^17.1.0",
```
Comment (scinos) [inline package.json]: it should be a `devDependency`.

Also, don't you need the peer dependencies? (https://www.npmjs.com/package/eslint-config-airbnb#eslint-config-airbnb-1)

File: .eslintrc.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+var OFF = 0, WARN = 1, ERROR = 2;
+
+module.exports = exports = {
+    "env": {
+        "es6": true
+    },
+
+    "extends": ["react-app", "airbnb"],
```
Comment (scinos) [inline .eslintrc.js]: What is `"react-app"`? I've never seen it.

File: package.json (hunk 4 above)
Comment (2940cristian) [reply package.json]: Thanks, added it to dev dependencies and will be visible in next commit. Create-React-App comes with all the dependencies for "eslint-config-airbnb" in react-scripts so no need to install manually.

File: .eslintrc.js (hunk 5 above)
Comment (2940cristian) [reply .eslintrc.js]: "react-app" is create-react-app's eslint configuration that comes from react-scripts. Without ejecting the only other way is to extend "react-app" and create your own rules or extend another such as "airbnb" 

https://github.com/facebook/create-react-app/issues/808

Comment (scinos) [reply .eslintrc.js]: TIL, thanks

**Pull request merged**